client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureForm.tsx

```diff
@@ -95,17 +141,13 @@ const ExposureForm = (props: Props) => {
 										}}
 										value={exposureSourceSearchString}
 										test-id='exposureSource'
-										onSearchClick={setOptionalCovidPatientsAsync}
-										onKeyDown={(e: React.KeyboardEvent) => {
-											if (e.key === 'Enter') {
-												e.preventDefault();
-												setOptionalCovidPatientsAsync()
-											}
-										}}
+										onSearchClick={() => {}}
```

Comment:
maybe you can remove: onKeyDown and onSearchClick

Reply:
fixed